File: responses/W3PT58-PJ8/sum_of_integers_in_a_string.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(A) {
    // write your code in JavaScript
    
    A.sort(function(a, b) { return a - b });
    
    var highestInt = null;
    var times = 0;
    
    for(i = 0; i < A.length; i++) {
        var currentInt = A[i];
        
        if(highestInt == null || highestInt != currentInt) {
            highestInt = currentInt;
            times = 1;
        } else {
            times++;
        }
    }
    
    return times;
}

File: responses/9CKBRD-ZZ8/sum_of_integers_in_a_string.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(A):
    
    highest_int = -99999999999999999
    highest_count = 0
    
    if len(A) == 0:
        return 0
    else:
        for integer in A:
            if integer > highest_int:
                highest_int = integer
                highest_count = 1
            elif integer == highest_int:
                highest_count += 1
    
    return highest_count
